test/test_tensorboard.py

```diff
@@ -131,6 +131,7 @@ def test_pytorch_histogram(self):
         with self.createSummaryWriter() as w:
             w.add_histogram('float histogram', torch.rand((50,)))
             w.add_histogram('int histogram', torch.randint(0, 100, (50,)))
+            w.add_histogram('bfloat16 histogram', torch.rand((50,)).to(torch.bfloat16))
```

Comment:
This could just be `torch.rand(50, dtype=torch.bfloat16)`, which is a bit shorter.

Reply:
Thanks for the suggestion. Fixed.